### DISCUSSION TITLE
Can't hide language in WakaTime

### DISCUSSION THREAD

**Discussion by DanielDekhtyar** [anuraghazra/github-readme-stats#3667] [Q&A] [answered]
Hi, one and all!

How can I hide Git Config from WakaTime stats?
aka &hide=  

![](https://github.com/anuraghazra/github-readme-stats/assets/44679647/d6cdd523-cbc8-4bb9-b44e-8d0eac2b38b4)

Thank you so much!

**Comment by qwerty541** [comment, marked as answer]
Hey, @DanielDekhtyar! You can hide git config from your card using `&hide=git%20config` query option. Value should be url encoded.

```html
<img src="https://github-readme-stats.vercel.app/api/wakatime?username=DanielDekhtyar&layout=compact&hide=JSON,Ezhil,Text,Other,git%20config" />
```

<img src="https://github-readme-stats.vercel.app/api/wakatime?username=DanielDekhtyar&layout=compact&hide=JSON,Ezhil,Text,Other,git%20config" />

**Reply by thorbm1500** [reply]
Thank you! I was struggling to hide "Image (SVG)", and this helped me as well.
